stories/Loader.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Story } from '@storybook/react';
+
+import { Loader } from '../src/components/atoms';
+import { DefaultProps as LoaderProps, LOADER_TYPE } from '../src/components/atoms/Loader/Loader';
```

Comment:
use Props instead of default props

Reply:
Loader do not have any Prop Type it only has default prop type